luigi/contrib/kubernetes.py

```diff
@@ -59,6 +59,9 @@ class kubernetes(luigi.Config):
     max_retrials = luigi.IntParameter(
         default=0,
         description="Max retrials in event of job failure")
+    kubernetes_namespace = luigi.Parameter(
```

Comment:
This should be an `OptionalParameter` if you want the default to be `None` and not show a warning

Reply:
Thanks, I've updated the branch.